src/ui/public/kuery/functions/is.js

```diff
@@ -38,28 +34,47 @@ export function toElasticsearchQuery(node, indexPattern) {
       }
     };
   }
-  else if (fieldName === '*' && value === '*') {
-    return { match_all: {} };
-  }
-  else if (fieldName === '*' && value !== '*') {
-    return {
-      multi_match: {
-        query: value,
-        fields: ['*'],
-        type: 'phrase',
-        lenient: true,
-      }
-    };
+
+  const fields = getFieldsByWildcard(fieldName, indexPattern);
+  const scriptedFields = fields.filter(field => field.scripted);
+  const nonScriptedFields = fields.filter(field => !field.scripted);
+  const queries = [];
+
+  if (value !== '*') {
```

Comment:
Exists query makes no sense for scripted fields, so skip this part if value is === '*'